### ISSUE TITLE
Formulaire JDMA : voir si les émojis posent problèmes

### ISSUE THREAD

**Issue by AnthonyBrunelli** [DISIC/wikidemarches#1066]

**Comment by AnthonyBrunelli** [comment]
Message reçu:

![image](https://user-images.githubusercontent.com/35567048/146392939-9650f9fe-e14d-46f5-9e4c-fe0342afb321.png)

**Comment by AnthonyBrunelli** [comment]
Effectivement, test effectué en prod par Joffrey sur une démarche de test. Après saisie d'émojis dans le champs en saisie libre "souhaitez-vous nous en dire davantage" : message d'erreur: 
![image](https://user-images.githubusercontent.com/35567048/150164632-e5bcd0e7-d7a1-4056-9c13-6b0043c347ac.png)

**Comment by AnthonyBrunelli** [comment]
Même problème sur _tous_ les champs en saisie libre (champs "autres")...

**Comment by AnthonyBrunelli** [comment]
Vu avec Anca: montée de version longue et coûteuse à cause de toutes les personnalisations (spécifiques) et base de données énormes.
Alternative: ne pas stocker les émojis, sans avoir un message d'erreur moche. Il faut bien signaler à l'utilisateur "Pas d'émoji".

**Comment by AnthonyBrunelli** [comment]
La mise à jour pourra être faite dans le cadre de la refonte.

**Comment by AnthonyBrunelli** [comment]
Question d' @ugodtn pour @lucaa : est-il possible de remplacer l'émoji par son unicode ? Par exemple en mettant : [emoji 1F600]
ça permettrait de ne pas perdre d'information et de ne pas avoir de blocage.

**Comment by lucaa** [comment]
> : est-il possible de remplacer l'émoji par son unicode ? Par exemple en mettant : [emoji 1F600]

ça doit être tout aussi facile que de les éliminer complètement, je dirais.

**Comment by revolunet** [comment]
apparement ce souci est toujours d'actualité et impacte toutes les démarches 😢 

je ne sais pas si mon avis a bien été pris en compte 

<img width="1215" alt="311515269-1e159e74-d32d-454f-beaf-580a365d4a48" src="https://github.com/DISIC/observatoire/assets/124937/0384a11c-5f40-4f98-b7ed-a43578cb0b39">


<img width="347" alt="311515280-d9b88bf9-a2a7-4e33-96e9-7a174badfe03" src="https://github.com/DISIC/observatoire/assets/124937/ca5d65fc-4ec7-4036-9a60-443cf6a85e04">

**Comment by maxime-lecoq** [comment]
Bonjour, 
Ce bug sera corrigé par la nouvelle plateforme qui sera mise en place cet été en production.
En vous remerciant,
Maxime Lecoq
PM JDMA
